content/updates/2017-06-12-release-notes.md

```diff
@@ -0,0 +1,25 @@
++++
+date = "2017-06-12"
+title = "Platform Release Notes: June 12, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past week and a half.
+<!--more-->
+
+### Fixed
+
+- We finished migrating all tenants from East/West into GovCloud! Thank you to all our customers for their patience and cooperation in making this move successful.
+- Updated tests, documentation and manual checks we perform to ensure we were ready to begin shutdown of the E/W environment.
```

Comment:
This is internal rather than customer-facing, so customers don't need to hear about it.